requirements_eaton.txt

```diff
@@ -0,0 +1,6 @@
+robotframework
```

Comment:
Per below, I don't actually think we will need this if we clean things up and run it properly.  Let me know if you continue to find issues.

Reply:
I totally remove this file. So, while executing I did not find any issues after removing this